google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceInfo.java

```diff
@@ -77,6 +78,14 @@ public abstract static class Builder {
 
     public abstract Builder setDisplayName(String displayName);
 
+    public Builder setUpdateTime(Timestamp updateTime) {
```

Comment:
Should we change the visibility of the `setUpdateTime` to protected? If required for testing, you could add VisibleForTesting annotation.
```suggestion
    @VisibleForTesting
    protected Builder setUpdateTime(Timestamp updateTime) {
```